salt/metalk8s/repo/installed.sls

```diff
@@ -8,6 +8,7 @@
 
 {%- set nginx_confd  = '/var/lib/metalk8s/repositories/conf.d/' %}
 {%- set nginx_default_conf = nginx_confd ~ 'default.conf' %}
+{%- set nginx_const_conf = nginx_confd ~ '99-registry-consts.inc' %}
```

Comment:
`constants` or something along those lines, not fond of `consts`

Reply:
I think I will use "common" like @gdemonet suggested